### ISSUE TITLE
Cartridge isn't updating with changes

### ISSUE THREAD

**Issue by AlexChesters** [icflorescu/openshift-cartridge-nodejs#10]
When I push my changes up to the cartridge, this is my output:

```
remote: CLIENT_RESULT: Application is already stopped.
remote: Building git ref 'master', commit some commit
remote: CLIENT_RESULT: Node.js modules installed.
remote: Preparing build for deployment
remote: Deployment id is some id
remote: Activating deployment
remote: CLIENT_MESSAGE: Starting Node.js application...
remote: CLIENT_RESULT: Node.js application started.
remote: -------------------------
remote: Git Post-Receive Result: success
remote: Activation status: success
remote: Deployment completed with status: success
```

However my changes aren't taking effect, in order for my changes to take affect I have to `ssh` onto the machine and manually kill and restart the node server.

**Comment by AlexChesters** [comment]
This is due to my change on [this line](https://github.com/icflorescu/openshift-cartridge-nodejs/blob/85fc08fb43f5eeb9e95f5eb4952dc2b1baf7061e/bin/control#L7). If I run that `ps -ef` on the box no output is returned, this is because the process is `node server` rather than `npm start`

**Comment by AlexChesters** [comment]
Suggestion, instead of searching for `npm start`, the script parses the `package.json`, grabs the command that is ran by `npm start` and `grep`s for that instead. I'll take a look at this now

**Comment by AlexChesters** [comment]
This has been fixed in #11
